.github/workflows/repository-management.yml

```diff
@@ -0,0 +1,250 @@
+---
```

Comment:
⛏️ We don't have to have these at the tops of workflows.

Reply:
We don't need them, but I believe it was decided a long time ago to keep them.  We can revisit that and make a card to update all the workflows across the organization to remove the document start marker, and then update the linter to make sure they don't exist at the top of the workflow files. From an initial search from the repos I have locallly cloned, it will be at least 209 workflow files across 30 repositories.

I do think we should settle on a standard and enforce it across all workflow files using the linter.

Reply:
I looked at it, and it's easy to make sure the linter catches document start and document end markers. I made a card and will put in a PR.